.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: android
-sudo: required
 jdk: oraclejdk8
-dist: trusty
```

Comment:
I don't see where `dist` is mentioned in that article

Reply:
Yes. Just taking advantage of this time to get rid of Trusty, which is on its last legs (EOL is April 2019).

Reply:
does Travis use the most recent available image if this is not set?

Reply:
Yes. At the moment this is still the Trusty image, but they are working on upgrading (this is part of the infrastructure consolidation). If we specify a version then they are constrained to use it.